### ISSUE TITLE
18439085: iOS 8 Today Widget - Voiceover skips contents of Widget

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#3082]
#### Description

Summary:
Voiceover skips the contents of third-party today widgets even if they have accessible content. Voiceover selects and reads off all the other elements in notification center (including first-party widgets like Today and Tomorrow), it even reads off the titles of the third-party widgets, but it will completely skip over their contents. The only way to get voiceover to speak the accessible elements in the widget is to touch inside the widget. But this will make it very hard if the user cannot see because they will not know where to touch.

Steps to Reproduce:
1. Open notification center.
2. If there aren't any third party widgets in the Today screen, add one.
3. Turn on voiceover.
4. Swipe right until voiceover selects all accessible elements on the screen.
5. Swipe back right all the way up to the top

Expected Results:
Voiceover should not skip the contents of the widget.

Actual Results:
Voiceover does skip the contents of the widget
## 

Product Version: 8.0 (12A365)
Created: 2014-09-24T12:31:03.854247
Originated: 2014-09-24T00:00:00
Open Radar Link: http://www.openradar.me/18439085
